jetty-http/src/main/java/org/eclipse/jetty/http/HttpURI.java

```diff
@@ -50,9 +50,29 @@
 {
     enum Ambiguous
     {
+        /**
+         * URI contains ambiguous path segments e.g. <code>/foo/%2e%2e/bar</code>
```

Comment:
Can you use the more modern `{@code /foo/%2e%2e/bar} instead?

Reply:
Done.